### DISCUSSION TITLE
Is there a way to add hyperlink to image/picture?

### DISCUSSION THREAD

**Discussion by deepk-kr** [ClosedXML/ClosedXML#2020] [Q&A]
1. I want to add a hyperlink to an image in a sheet on, pointing to another sheet within the workbook. Is there a way to do it?

I checked the wiki page for hyperlinks, there is no mention of image/picture.

2. I'm working with a template, from which I generate excel. Template contains images with hyperlink, on saving it. The hyperlinks gets removed. Why is it happening?

**Comment by jonathanIvorBetts** [comment]
If you're positioning the picture at a specific location you could try creating a named range in the same location then apply hyperlink to that?
